content/en/logs/workspaces/sql_reference.md

```diff
@@ -0,0 +1,58 @@
+---
+title: SQL Reference
+further_reading:
+- link: "/logs/workspaces/"
+  tag: "Documentation"
+  text: "Learn more about Log Workspaces"
+---
+
+## Overview
+
+SQL functions are built-in operations in SQL (Structured Query Language) used for tasks like calculations, data manipulation, or formatting. This documentation is a reference for SQL support available in Logs Workspaces. It details:
+- [SQL functions](#functions)
+- [Syntax compatible with PostgreSQL](#syntax)
+
+It provides a resource for the Analysis cell. Use this information to leverage SQL capabilities in data analysis and manipulation within Log Workspaces.
```

Comment:
While reading the doc, I found the following line a bit out of context.
> It provides a resource for the Analysis cell.

Should we not be leading with this? That the documentation provides a reference for the capabilities of the SQL Analysis cell?